content/general/incoming-webhooks.textile

```diff
@@ -0,0 +1,104 @@
+---
+title: Incoming Webhooks
+section: general
+index: 19
+languages:
+  - javascript
+meta_description: "Incoming webhooks provide a way to integrate external web services with Ably."
+meta_keywords: "Ably, incoming, inbound, webhooks, webhook configuration, web services, realtime"
+jump_to:
+  Help with:
+    - Configuring your incoming webhook#configure
+    - Test your incoming webhook#test
+    - Receiving the incoming webhook#receive
+    - Specifying optional headers#headers
+    - Summary#summary
+    - See also#see-also
+redirect_from:
+---
+
+Incoming webhooks provide a way to integrate external web services with Ably. The incoming webhooks are routed to Ably channels as "messages":/realtime/messages.
```

Comment:
I think this paragraph is misleading, in that it implies it is necessary to configure a webhook to make this possible. In fact, these incoming webhooks are just a convenience feature - it's perfectly possible to configure a webhook with an existing key - and I think that should be clear.